internal/serve/serve.go

```diff
@@ -266,6 +266,9 @@ func handleHTTP(o ServeOptions) *chi.Mux {
 			r.With(middleware.AnyRoleMiddleware(authManager, data.OwnerUserRole, data.FinancialControllerUserRole)).
 				Post("/", handler.PostDisbursement)
 
+			r.With(middleware.AnyRoleMiddleware(authManager, data.OwnerUserRole, data.FinancialControllerUserRole)).
+				Delete("/{id}", handler.DeleteDisbursement)
```

Comment:
Could you plz include this endpoint to the Test_handleHTTP_authenticatedEndpoints test in serve_tests.go?

Reply:
Good point. Addressed in 38e26676145c2ccfd8236a675b54610405d0e106